x-pack/plugins/reporting/server/lib/store/store.ts

```diff
@@ -52,7 +77,7 @@ export class ReportingStore {
     this.indexPrefix = config.get('index');
     this.indexInterval = config.get('queue', 'indexInterval');
     this.logger = logger.clone(['store']);
-    this.queueTimeoutMins = Math.ceil(numberToDuration(config.get('queue', 'timeout')).asMinutes());
+    this.queueTimeoutSecs = Math.ceil(numberToDuration(config.get('queue', 'timeout')).asSeconds());
```

Comment:
This change fixes an issue where the queue timeout could not be less than 1 minute